docs/manual/releasing.rst

```diff
@@ -0,0 +1,47 @@
+===============
+Releasing Qtile
+===============
+
+Here is a short document about how to tag Qtile releases.
+
+1. create a "Release vX.XX.XX" commit. I like to thank all the contributors; I
+   find that list with something like:
+
+.. bash::
+
+   git log --oneline --no-merges --format=%an $(git describe --tags --abbrev=0)..HEAD | sort -u
+
+note that this can sometimes generate duplicates if people commit with slightly
+different names; I typically clean it up manually.
+
+Be sure that you GPG-sign (i.e. the ``-S`` argument to ``git commit``) this commit.
+
+2. Create a GPG-signed annotated tag (``git tag -a -s vX.XX.XX``); I usually just use
+   exactly the same commit message as the actual release commit above.
+
+3. Push your tag to qtile/qtile directly:
+
+.. bash::
+
+   git push origin vX.XX.XX
```

Comment:
For me, `origin` is my fork, `upstream` is the main qtile repo...

Reply:
This is the risk of writing explicit git commands :). Should I drop them? Or what do you prefer?

Reply:
Maybe remove the example and be more explicit in line 22 e.g. adding something like `... as opposed to your fork`